### PR TITLE
User being able to see item after being retired

### PR DESCRIPTION
**Pull request by bryceholcomb**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -30,6 +30,7 @@ gem 'unicorn'
 gem 'cancan'
 gem 'rmagick'
 gem 'carrierwave'
+gem 'unscoped_associations'
```
Comment (kpearson) [inline Gemfile:33]: What is this beauty?

File: app/controllers/items_controller.rb [hunk 2]
```diff
@@ -4,6 +4,6 @@ def index
   end
 
   def show
-    @item = Item.find(params[:id])
+    @item = Item.unscoped.find(params[:id])
   end
```
Comment (kpearson) [inline app/controllers/items_controller.rb:8]: unscope that!

**Comment by scottalexandra** [comment]
looks wonderful!

File: Gemfile (hunk 1 above)
Comment (bryceholcomb) [inline Gemfile:33]: Its a life saver if you scope any defaults

**Pull request merged**